### DISCUSSION TITLE
"Shift-click" not working on "Single mouse button mode"

### DISCUSSION THREAD

**Discussion by RandomDude90** [runelite/runelite#14944] [Q&A]
The "Menu Entry Swapper"  "Customizable shift-click"  doesn't seem to be working when using "Single mouse button mode" from the settings, I've confirmed this with a few players. They all seem to be facing the same issue. 

It works fine when not using "Single mouse button mode".

I'm not sure when the problem began as I haven't used "Single mouse button mode" for some time. 

**Environment**
 - OS:  Windows 10
 - RuneLite version: 1.8.18.2
 - Launcher version: 2.3.0

**Comment by LlemonDuck** [comment]
I'm not sure what the expectation here is. Shift-click on single mouse mode seems to be prioritizing the correct options for me, but it will still require the second click to activate, like all actions in single mouse mode.

Are you hoping for shift-clicking to bypass the need for a second click?

**Reply by geheur** [reply]
Bypassing that second click is generally what people are asking for with this, yeah. I've seen people claim in the discord that it used to be possible to bypass the right-click menu with shift-click at some point in the past, even while in single mouse button mode.

FWIW, the one usecase I've heard of this is using the no mouse movement thieving method (right-clicking near the bottom of the game client so the right-click menu opens with "pickpocket" already under the mouse cursor) with single mouse button mode so that you can just spam left-click to thieve the npc. However you can take damage from failing pickpockets so food has to be eaten, which is made more difficult with single mouse button mode, and would be made easier if you could shift-click to "Eat" without opening the menu.
